e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -229,20 +230,24 @@ export function waitForSyncToFinish({
     if (!body.tables.length) {
       waitForSyncToFinish({ iteration: ++iteration, dbId, tableName });
     } else if (tableName) {
-      const hasTable = body.tables.some(
+      const table = body.tables.find(
         table =>
           table.name === tableName && table.initial_sync_status === "complete",
       );
-      if (!hasTable) {
+      if (!table) {
         waitForSyncToFinish({ iteration: ++iteration, dbId, tableName });
       }
+
+      if (tableAlias) {
+        cy.wrap(table).as(tableAlias);
```

Comment:
This is necessary to get access to the created table, so I could use their info in the test e.g. table ID, field IDs